### DISCUSSION TITLE
Domain says invalid configuration when everything is correct?

### DISCUSSION THREAD

**Discussion by samuel129** [vercel/community#3158] [Bug Report]
### Page to Investigate

https://vercel.com/samuel129/netflix-copycat/settings/domains

### Steps to Reproduce

My website that was hosting correctly for the past couple of weeks started to stop working, when I checked the domains page I noticed that my site: (www.samuel-kim.com) now says "Invalid Configuration.

The DNS records that the site asked me to add were already on the configurations.

https://i.imgur.com/RG3DjIz.png
https://i.imgur.com/4w2wryy.png

**Comment by smaeda-ks** [comment]
taking a look

**Reply by smaeda-ks** [reply]
@samuel129 it looks like your domain was shut down due to a suspicion of hosting a phishing website. Could you please explain what exactly you were trying to build?

**Reply by samuel129** [reply]
Hey, thanks for the help, I completed a netflix clone website for my portfolio and wanted to host it on Vercel to show my future employers.

Is there any workaround for this?

**Reply by smaeda-ks** [reply]
@samuel129 You really should consider adding some sort of authentication layer to only allow people you trust to access such content.

See our deployment protection feature:
- https://vercel.com/docs/concepts/deployments/deployment-protection

Otherwise, it's also easy to implement a basic auth using Edge Middleware:
- https://github.com/vercel/examples/tree/main/edge-middleware/basic-auth-password

This way, you can avoid some random page visitors being confused by the clone website and may report it as a phishing website, causing an issue like this. And in general, please make it clear that your website is created for a learning/exercise/demo purpose and not trying to actually accept any user inputs/submissions for the login form, for instance. You shoud add a sticky page header component or as such.

**Reply by smaeda-ks** [reply]
I'm also contacting our upstream provider (domain registrar) to unblock your domain. Will keep you updated.

**Comment by samuel129** [comment]
Awesome thank you so much for your help, ill make sure to add
authentication as well as a component showing that its a demo site :)
